File: LKH/1주차/1284.py
n = int(input())
i = 2
ans = []

while n!=1:
    if n% i ==0:
        n = n/i
        ans.append(i)
    else:
        i +=1

if len(ans)==2:
    print(ans[0], ans[1])
else:
    print("wrong number")

File: SJH/1284.cpp
#include<iostream>
#include<cmath>
using namespace std;

int main() {
	int a;
	int big;
	int small;
	int num=0;

	cin >> a;

	for (int i = 2; i < a; i++) {
		if (a%i == 0) {
			num++;
			if (num ==1) {
				small = i;
			}
			else if (num == 2) {
				big = i;
			}
		}
		if (num == 3) {
			break;
		}
	}

	if (num == 2) {
		if (pow(small, 2) == big) {
			cout << "wrong number";
		}
		else {
			cout << small << " " << big;
		}
	}
	else {
		cout << "wrong number";
	}


	return 0;
}